airbyte-integrations/connectors/source-oura/source_oura/spec.yaml

```diff
@@ -0,0 +1,34 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Oura Spec
+  type: object
+  required:
+    - api_key
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      description: API Key
+      airbyte_secret: true
+      order: 0
+    start_date:
+      type: string
+      description: Start date
+      pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
+      order: 1
+    end_date:
+      type: string
+      description: End date
+      pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
+      order: 2
+    start_datetime:
+      type: string
+      description: Start datetime for heart rate data in ISO 8601 format
+      pattern: ^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(Z|[+-](?:2[0-3]|[01][0-9]):[0-5][0-9])?$
+      order: 3
+    end_datetime:
+      type: string
+      description: End datetime for heart rate data in ISO 8601 format
+      pattern: ^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(Z|[+-](?:2[0-3]|[01][0-9]):[0-5][0-9])?$
+      order: 4
```

Comment:
Definitely not a good pattern here. Please use string interpolation to extract the date from datetime.
Start Datetime is required?

Reply:
Good point. Have become more familiar with the low code CDK over the weekend, so much better with it now.